### DISCUSSION TITLE
Are Crashlytics and Analytics available in SPM for tvOS?

### DISCUSSION THREAD

**Discussion by melvitax** [firebase/firebase-ios-sdk#8609] [Q&A] [answered]
I'm not 100% clear based on the Swift Package Manager read me. Are Crashlytics and Analytics available for tvOS when using Swift Package Manger?

**Comment by paulb777** [comment, marked as answer]
Crashlytics is available via community support for tvOS. Analytics is not. See https://firebase.google.com/docs/ios/learn-more#firebase_library_support_by_platform and #2011

Update: as of version 8.9.0, Analytics is available on tvOS.

**Reply by melvitax** [reply]
Thanks @paulb777 for the quick response!
